metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
The reason for millis is that nanos would have resulted into too human unfriendly values to show in Kibana. Best would be nanos + field formatter in Kibana, but it wasn't supported then and I'm not sure it is now. 

Since timelion is meant to be main consumer for this, we should check how good/bad that works in timelion at the moment. Long term, we plan to use mappings for defining the unit, right?

`response_time` is better than `responsetime` in a trivial way that IMO doesn't justify losing consistency or doing backwards incompatible changes.